spec/models/viewing_party_spec.rb

```diff
@@ -14,5 +14,35 @@
     it { should validate_presence_of :time }
     it { should validate_presence_of :user_id }
     it { should validate_presence_of :movie_id }
+
+    it 'should only accept dates > today' do
+      user = User.create(
+        email: 'testing@example.com',
+        password: '1234**USAusa',
+        password_confirmation: '1234**USAusa'
+      )
+      movie = Movie.create(title: 'fight club', uuid: 550)
+
+      vp = user.viewing_parties.new(movie_id: movie.id, duration: 100, date: '2018-01-01', time: Time.now, created_at: Time.now, updated_at: Time.now)
+      expect(vp.save).to eq(false)
+      vp = user.viewing_parties.new(movie_id: movie.id, duration: 100, date: Date.today, time: Time.now, created_at: Time.now, updated_at: Time.now)
+      expect(vp.save).to eq(true)
+    end
+
+    it 'should only accept duration >= 5' do
```

Comment:
Is this a time limit of 5 hours for the party? Was that just arbitrary ...

Reply:
5 minutes, but yes a fairly arbitrary number.   Just want to make sure it's not negative.